src/Pages/Reconstruction.jsx

```diff
@@ -43,28 +60,57 @@ export class Reconstruction extends Component {
         },
       ],
     };
-    this.changeNameHandler = this.changeNameHandler.bind(this);
     this.submitHandler = this.submitHandler.bind(this);
-    this.onFileChange = this.onFileChange.bind(this);
+    this.clickHandler = this.clickHandler.bind(this);
   }
 
-  changeNameHandler(e) {
+  clickHandler() {
+    this.form.current.reset();
```

Comment:
We might need to check if current is a valid object.
```suggestion
    this.form.current && this.form.current.reset();
```